client/sites/components/sites-dataviews/index.tsx

```diff
@@ -24,10 +24,13 @@ type Props = {
 	dataViewsState: View;
 	setDataViewsState: ( callback: ( prevState: View ) => View ) => void;
 	selectedItem: SiteExcerptData | null | undefined;
-	openSitePreviewPane: (
-		site: SiteExcerptData,
-		source: 'site_field' | 'action' | 'list_row_click' | 'environment_switcher'
-	) => void;
+	sitePreviewPane: {
+		open: (
+			site: SiteExcerptData,
+			source: 'site_field' | 'action' | 'list_row_click' | 'environment_switcher'
+		) => void;
+		getUrl: ( site: SiteExcerptData ) => string;
+	};
```

Comment:
Is it worth extracting this type to a common place or somehow re-using the same type of def from the other file?

Reply:
Good point, done in 7e424a112a0ee9d1512ddd3b28d357abef5f2ee8